SRT to VTT Converter/MainWindow.xaml.cs

```diff
@@ -211,8 +211,8 @@ private void BackgroundWorker_RunWorkerCompleted(object sender, RunWorkerComplet
 
 		private void Convert(string sFilePath)
 		{
-			using (var strReader = new StreamReader(sFilePath))
-			using (var strWriter = new StreamWriter(sFilePath.Replace(".srt", ".vtt")))
+			using (var strReader = new StreamReader(sFilePath, Encoding.GetEncoding(1256)))
+			using (var strWriter = new StreamWriter(sFilePath.Replace(".srt", ".vtt"), true))
```

Comment:
Setting the `append` parameter to `true` doesn't make sense since you said these changes will "replace the vtt file when we reCreate it" (which is what the program already does). Was this intentional?